django_tenants/staticfiles/finders.py

```diff
@@ -8,6 +9,39 @@
 from django_tenants import utils
 from django_tenants.staticfiles.storage import TenantStaticFilesStorage
 
+class TenantAppDirectoriesFinder(AppDirectoriesFinder):
+    """
+    This is a replacement for the AppDirectoriesFinder. The only difference between
+    this and the standard AppDirectoriesFinder implementation is that this one will
+    understand how tenant-aware file handling works and where to search for your
+    files.
+
+    In-detail explanation:
+
+    Let's say that we have a Tenant called "Foo" (foo.com).
+    When we try to access, let's say, foo.com/admin, the browser will issue a request
+    to (among others) foo.com/static/foo/admin/css/base.css , instead of the
+    usual request if we didn't use django-tenants (foo.com/static/admin/css/base.css).
+
```

Comment:
My suggestion as part of #265 and #266 is that django-tenant's ``TenantStaticFilesStorage `` should not be injecting the ``schema_name`` in static URLs (thereby breaking any standard downstream finders).

Instead, if the tenant-aware finders fail to locate a static file, then the standard finders should serve as fallback.

Reply:
Let me try this, but I think there will be problems once the project is deployed and server by uWSGI.